modules/converting-machine-api-infra-machines.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="converting-machine-api-infra-machines_{context}"]
+= Converting infrastructure machines created using the Machine API
+
+[IMPORTANT]
+====
+Creating infra machines using the Machine API is now deprecated.
+====
+
+Infrastructure machines created using the Machine API had both the `worker` and `infra` labels applied. This duel label required you to taint the node so that compute workloads were not scheduled on it. Managing infra workloads using taints is no longer recommended. Machines created using the Machine API must be converted to use only the `infra` label and have a custom machine config pool manage it.
```

Comment:
A few suggestions. 

Change: "This duel label required you to taint the node so that compute workloads were not scheduled on it" to "This duel label required users to also taint the node so that compute workloads were not scheduled on it, to stay in compliance with our subscription guidelines". 

Change: "Managing infra workloads using taints is no longer recommended." to "Managing infra workloads using taints is a tedious and cumbersome process, which is no longer recommended, for infra nodes." The 'for infra nodes part of this is critical, because we don't want to confuse customers with this statement, by suggesting tainting nodes is not recommended. 

Change: "Machines created using the Machine API must be converted to use only the `infra` label and have a custom machine config pool manage it" to "Machines created using the Machine API must be converted to use only the `infra` label and have a custom machine config pool which manages it".

Reply:
also "duel" -> "dual"

Reply:
@wking good call :sweat_smile: . We don't want people to think the two labels are dueling.

Reply:
@yuqi-zhang have we validated what is the defaultNodeSelector value for the scheduler? IIRC I do not think it uses the worker label by default, or specifically anything.  So we would need to add documentation explaining that need to set worker as the defaultNodeSelector, otherwise customer workloads will still get scheduled to infra nodes. (https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default)

Reply:
@yuqi-zhang have we checked the above?  the latest cluster I ran on AWS had no defaultNodeSelector value, meaning that even if we label infra nodes in a particular way, customer workloads can still land on them.